web/packages/teleport/src/Locks/Locks.tsx

```diff
@@ -43,6 +44,20 @@ function getFormattedDate(d: string): string {
   }
 }
 
+function lockTargetsMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof LockForTable & string
+) {
+  if (propName === 'targets') {
+    return targetValue.some(
+      ({ name, value }) =>
+        name.toUpperCase().includes(searchValue) ||
```

Comment:
Shouldn't `searchValue` also be `toUpperCase()`'ed?

Reply:
searchValue comes in uppercased already

Reply:
For future readers, it happens here: https://github.com/gravitational/teleport/blob/master/web/packages/design/src/utils/match/match.ts#L43